gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -1104,23 +1104,27 @@ function GM:OnContextMenuOpen()
 	end
 end
 
+function TTT2CacheEquipMaterials(item)
+	--if there is no material or model, the item should probably not be available in the shop
+	if item.material and item.material ~= "vgui/ttt/icon_id" then
+		item.ttt2_cached_material = Material(item.material)
```

Comment:
while you're at it, you should probably change the name in the TTT2 project. The material is names `.iconMaterial` for roles and I think it should be the same here. Also the model should have a smiliar name,

Reply:
![grafik](https://user-images.githubusercontent.com/72046466/128041996-32c784f6-8eb4-4d97-9afb-25ac4264d5e3.png)
It would be material. But I could change that.

Reply:
Your ttt2 Items do use material though

Reply:
> Your ttt2 Items do use material though

What do you mean?

Reply:
![grafik](https://user-images.githubusercontent.com/72046466/128042830-e1ea22f1-0bed-40f5-b233-ba0b7d0dbc38.png)

Reply:
ah, but this is a path, not a real material. Bad name

Reply:
So, I dont really know what this would do, if I change it. TTT2 only uses it in some places, so maybe do it in another refactor, so that this change doesnt introduce new bugs. ^^

Reply:
Don't change it for now, this might break compatibility to other addons